CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Mapbox welcomes participation and contributions from everyone.
 
 ### master
 
+### 5.9.0-alpha-1 - February 12, 2021
+- Fixed DirectionRouteRefresh flow [#1225](https://github.com/mapbox/mapbox-java/pull/1225)
```

Comment:
```suggestion
- Fixed `DirectionsRouteRefresh` flow [#1225](https://github.com/mapbox/mapbox-java/pull/1225)
```

Reply:
Could you mention the motivation for this change and why it's a breaking one?

Reply:
```Fixed `DirectionsRefreshResponse` flow: introduced `DirectionsRouteRefresh` and restricted dependency on `DirectionsRoute` [#1225](https://github.com/mapbox/mapbox-java/pull/1225)```

/cc @LukasPaczos

Reply:
@LukasPaczos Could you confirm please line above is good enough

Reply:
> Fixed `DirectionsRefreshResponse` flow: introduced `DirectionsRouteRefresh` and restricted dependency on `DirectionsRoute` which is a critical bug fixes. Even though this is a semver-breaking change, we're not considering it as one because the refresh feature was non-functional beforehand. [#1225](https://github.com/mapbox/mapbox-java/pull/1225)

Reply:
that's one of the longer changelog note that I've ever seen 😄